tests/test_cvt.py

```diff
@@ -77,16 +77,16 @@ def test_cvt_qnb(mesh, ref):
     assert_norm_equality(m.points, ref, 1.0e-9)
 
 
-def test_cvt_qnb_boundary(n=10):
+def test_cvt_qnb_boundary(n=50):
```

Comment:
`n=10` plays nicer with tests.

Reply:
`n=50` looks more like a circle. But let me revert it to `n=10`.

Reply:
Done.